.github/labeler.yml

```
@@ -0,0 +1,15 @@
+system:
```

yamllint

[error] wrong new line character: expected \n

(new-lines)